### PR TITLE
feat(orc-649): add basic review instructions for copilot

### PR DESCRIPTION
**Pull request by chasingrainbows**
This pull request introduces a comprehensive set of GitHub Copilot and code review instructions for the Lido Oracle project. The new documentation enforces strict standards around security, oracle logic correctness, testing, documentation updates, and Python code quality. These guidelines are intended to ensure the safety, reliability, and maintainability of code in a high-stakes DeFi environment.

## Related Issue/Task
- Related task: orc-649
- Epic: srv3

## How Has This Been Tested?
Describe how you tested the changes:
- [ ] Local tests (e.g., `pytest`)
- [x] Manual testing (describe steps)
- [ ] Not tested (explain why)

## Checklist
- [ ] Documentation updated (if required)
- [ ] New tests added (if applicable)
- [ ] `CSM_STATE_VERSION` is bumped (if the new version affects data in the cache)

### CODE REVIEW

File: .github/instructions/security.instructions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Private Key Security (CRITICAL)
+- **BLOCK**: `MEMBER_PRIV_KEY` must NEVER appear in logs, print statements, or error messages
+- **REQUIRE**: Private key variables only in `PRIVATE_ENV_VARS` dict in `src/variables.py`
+- Check signature operations maintain cryptographic security standards
+
+## Smart Contract Interactions
+- **REQUIRE**: For state-changing operations, verify the code preserves existing membership/role/reportability checks before sending a transaction
+- Check transaction parameters use proper gas estimation (prevent DoS)
+- Ensure contract addresses are checksummed and validated against known deployments
```
Comment (F4ever) [inline .github/instructions/security.instructions.md]: Maybe remove this?

Comment (F4ever) [reply .github/instructions/security.instructions.md]: Add item that we should double check types in responses from different nodes or if this is string, make sure, we always lower case or we are just sure it's case sensitive

File: .github/copilot-instructions.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- `accounting`: main protocol accounting oracle. Builds frame-based protocol reports, updates TVL-related state, processes withdrawal/finalization inputs, controls bunker-related values, and submits extra data
+- `ejector`: validator exit oracle. Determines which validators should exit to satisfy withdrawal demand and reports exit data on-chain
+- `csm`: Community Staking Module oracle. Consumes performance data, publishes report artifacts to IPFS, and reports performance/fee data for the community staking module
+- `cm`: Curated Module oracle. Similar staking-module reporting flow for curated staking modules
+- `check`: preflight and health-check module that validates environment and external dependencies before running an oracle
+- `performance_collector`: sidecar that ingests consensus-layer attestation/performance data and stores it in Postgres
+- `performance_web_server`: sidecar API serving performance data to staking-module oracles
+
+## Architecture Notes
+- Oracle code reads state from both Execution Layer and Consensus Layer, then builds deterministic frame-based reports for contracts
```
Comment (F4ever) [inline .github/copilot-instructions.md]: Reads data from KAPI also, and side cars if this is csm

File: .github/copilot-instructions.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## CI Context
+- Avoid flagging issues already enforced by CI unless the change clearly bypasses or weakens that protection
+- CI already runs `pytest -m 'not fork' tests`, `ruff format --check tests`, `ruff check` for changed Python files, `pyright src`, Docker image build, and Docker reproducibility checks
+- Prefer review comments about missing tests only when behavior, safety checks, protocol invariants, or integration boundaries changed
+
+## Key Patterns
+- Secrets belong only in private env handling and must never be logged
+- Changes around `ref_slot`, finalized slots, report phases, bunker mode, or transaction sending deserve extra scrutiny
+
+## Review Approach
```
Comment (F4ever) [inline .github/copilot-instructions.md]: duplicated in Review Philosophy
